site/src/pages/CreateTokenPage/CreateTokenForm.tsx

```diff
@@ -116,6 +116,7 @@ export const CreateTokenForm: FC<CreateTokenFormProps> = ({
 
             {lifetimeDays === "custom" && (
               <TextField
+                data-chromatic="ignore"
```

Comment:
Not sure what this is :thinking:

Reply:
someone let a leaky date sneak in to chromatic again, so I'm muting it 😅